adsmp/app.py

```diff
@@ -39,9 +88,13 @@ def update_storage(self, bibcode, type, payload):
                 r.orcid_claims = payload
                 r.orcid_claims_updated = now
             elif type == 'fulltext':
-                oldval = r.fulltext
+                oldval = 'not-stored'
                 r.fulltext = payload
                 r.fulltext_updated = now
+            elif type == 'metrics':
+                oldval = 'not-stored'
+                r.metrics = payload
+                r.metrics_updated = now
             else:
                 raise Exception('Unknown type: %s' % type)
```

Comment:
Since the previous suggestion for converting 'raise Exception' to log was well received, I will note the other lines with raise.  Perhaps they could also be log.